examples/asset-management/config.env

```
@@ -0,0 +1,8 @@
+SYNQLY_ORG_TOKEN=<synqly_api_token>
+SYNQLY_SERVICENOW_URL=<synqly_servicenow_url>
+SYNQLY_SERVICENOW_TOKEN=<synqly_servicenow_token>
+SYNQLY_ARMIS_URL=<synqly_armis_url>
+SYNQLY_ARMIS_TOKEN=<synqly_armis_token>
+SYNQLY_NOZOMI_URL=<synqly_nozomi_url>
+SYNQLY_NOZOMI_USER=<synqly_nozomi_user>
+SYNQLY_NOZOMI_TOKEN=<synqly_nozomi_token>
```

to be consistent with the way Camilo did this, let's change to

```suggestion
SYNQLY_ORG_TOKEN=
SYNQLY_SERVICENOW_URL=
SYNQLY_SERVICENOW_TOKEN=
SYNQLY_ARMIS_URL=
SYNQLY_ARMIS_TOKEN=
SYNQLY_NOZOMI_URL=
SYNQLY_NOZOMI_USER=
SYNQLY_NOZOMI_TOKEN=
```